conf/bridge-server.conf

```diff
@@ -1,7 +1,7 @@
 synapse.user = yours-synapse-user
 synapse.api.key = yours-synapse-api-key
 
-exporter.synapse.id = 3325672
+exporter.synapse.id = exporter-synapse-id
```

Comment:
We should probably remove this from this change. Don't want to add extra unnecessary stuff. Especially if it will generate more work for configuring Heroku.